src/components/Form/Visibility.js

```diff
@@ -9,17 +9,30 @@ function Visibility({
   control,
   errors,
   label = 'Who can see this entry?',
+  reduceAssistantOptions = false,
```

Comment:
Rather than adding a parameter, my suggestion would be to have this Visibility control get the necessary info via useUserStore, so the other forms don't need to worry about it. Similar to how it currently gets the site visibility options via useSiteStore

Reply:
I have updated the PR to handle the visibility options in the visibility control form since I no longer need to set the defaults in the other forms. Thanks for the suggestion!